32/screamingskulls/challenge.py

```diff
@@ -0,0 +1,10 @@
+from selenium import webdriver
+
+
+def main():
+    driver = webdriver.Chrome()
+    driver.get('http://pyplanet.herokuapp.com/')
```

Comment:
where are the tests? (lol) - feel free to write some test code as per instructions: https://codechalleng.es/challenges/32/